src/views/Todo-Components/Todos.js

```diff
@@ -46,6 +48,14 @@ const Todos = () => {
     setTodo("");
   };
 
+  const onCheckBoxClick =(data)=>{

```

Comment:
Instead of passing down whole data object, we can simply pass id. Same goes for Edit button as well.